src/Resources/config/services.yml

```diff
@@ -9,7 +9,7 @@ services:
         arguments:
             - "@service_container"
         tags:
-            - { name: kernel.event_listener, event: kernel.controller, method: onKernelController, priority: 254 }
+            - { name: kernel.event_listener, event: kernel.controller, method: onKernelRequest, priority: 31 }
```

Comment:
I assume you have set the priority to `31` on purpose. Can you elaborate please?